recipe/meta.yaml

```
@@ -34,9 +34,11 @@ requirements:
 
 test:
   commands:
-    - test -f ${PREFIX}/lib/librocksdb.a
-    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
-    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
+    - test -f ${PREFIX}/lib/librocksdb.a        # [not win]
+    - test -f ${PREFIX}/lib/rocksdb.lib         # [win]
+    - test -f ${PREFIX}/lib/librocksdb.dylib    # [osx]
+    - test -f ${PREFIX}/lib/librocksdb.so       # [linux]
+    - test -f ${PREFIX}/lib/rocksdb-shared.lib  # [win]
     - test -f ${PREFIX}/include/rocksdb/db.h
```

Windows needs CMD syntax
```suggestion
    - test -f ${PREFIX}/lib/librocksdb.a                   # [unix]
    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}         # [unix]
    - test -f ${PREFIX}/include/rocksdb/db.h               # [unix]
    - if not exist %LIBRARY_PREFIX\lib\rocksdb-shared.lib  # [win]
    - if not exist %LIBRARY_PREFIX\include\rocksdb\db.h    # [win]
    - if not exist %LIBRARY_PREFIX%\librocksdb.lib         # [win]
```